src/admin/components/NavBar/index.js

```diff
@@ -2,16 +2,19 @@ import React, { PropTypes } from 'react';
 
 import {
   NavBar,
+  NavBarButton,
   NavBarLink,
 } from 'core/components/NavBar';
 
-const AdminNavBar = () => (
+const AdminNavBar = ({ authenticated, logOut }) => (
```

Comment:
This pure function pattern is nice since this will blow up when the require props aren't supplied.